open-machine-learning-jupyter-book/deep-learning/nn.ipynb

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
<p>I think it would be better for the individual components to be bolded in *** *** to emphasize them instead of using subheadings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/697/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
done